### PR TITLE
Added Github actions for testing and linting on windows,ubuntu, macos (closes#2) 

### PR DESCRIPTION
**Pull request by aikido-sensei**

### CODE REVIEW

File: unit_tests/multiply_two_numbers.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+def multiply_2_nums(num1: int, num2: int) -> int:
+    """
+
```
Comment (silvs110) [inline unit_tests/multiply_two_numbers.py]: Add method's description (i.e what it does)

File: unit_tests/multiply_two_numbers.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    :param num1: int : Number 1 used in the multiplication
+    :param num2: int: Number 2 used in the multiplication
```
Comment (silvs110) [inline unit_tests/multiply_two_numbers.py]: To specify a parameters type use `: type param_name: `

Example:
```
: param num1: Number 1 used in the multiplication.
: type num1: int
```

File: unit_tests/multiply_two_numbers.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    :return: int: The result of the multiplication of num1 and num2
```
Comment (silvs110) [inline unit_tests/multiply_two_numbers.py]: Use `rtype` to describe return type. Example:

```
:return: The result of the multiplication of num1 and num2.
: rtype: int
```
Also, if the function raises an exception, add it to the docstring.  This [article](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html?highlight=domains#info-field-lists) gives you more information,

File: unit_tests/multiply_two_numbers_test.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import unittest
+from unit_tests.multiply_two_numbers import multiply_2_nums
+
+
+class MultiplyTwoNumsTest(unittest.TestCase):
+
+    def test_multiplication(self):
+        """
```
Comment (silvs110) [inline unit_tests/multiply_two_numbers_test.py]: Add test description

**Review by silvs110 — CHANGES_REQUESTED** [review]
Please change Pull Request title to the main commit title. And also link the issue2 to this PR by changing your commit's format:
```
<Title>

closes#<issue_number>
<description>
```

**Review by the-c0d3br34k3r — CHANGES_REQUESTED** [review]
Please be a bit verbose about the parameters used in the commands.

File: workflows/MacOs_build.yml [hunk 5]
```diff
@@ -0,0 +1,34 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python on Mac OS
```
Comment (the-c0d3br34k3r) [inline workflows/MacOs_build.yml]: Please rename file to macos_build.yaml

**Review by the-c0d3br34k3r — CHANGES_REQUESTED** [review]
Reviewed. Requested changes.

**Comment by the-c0d3br34k3r** [comment]
Please move the workflows folder to the .github folder.

**Comment by the-c0d3br34k3r** [comment]
Add these actions to the .github repository and then set it up for this repo.

**Comment by aikido-sensei** [comment]
> Add these actions to the .github repository and then set it up for this repo.

done

**Review by the-c0d3br34k3r — CHANGES_REQUESTED** [review]
Can we combine the different files for each OS into a single file like here?
https://github.com/padaiyal/jMavenProjectTemplate/blob/main/.github/workflows/maven_build.yml

**Comment by aikido-sensei** [comment]
> Can we combine the different files for each OS into a single file like here? https://github.com/padaiyal/jMavenProjectTemplate/blob/main/.github/workflows/maven_build.yml

Done! I left the other files for the moment in case there is an issue with the new one.

**Pull request closed without merging**